ovirt-release-host-node.spec.in

```diff
@@ -180,7 +180,8 @@ systemctl restart cockpit.service >/dev/null 2>&1
 
 PYTHON=$(command -v python3 || command -v python)
 
-for REPO in %{_sysconfdir}/yum.repos.d/CentOS-oVirt-*.repo;
+pushd %{_sysconfdir}/yum.repos.d
+for REPO in $(find -maxdepth 1 -type f -name 'CentOS-oVirt-*.repo' -printf '%f\n');
```

Comment:
So AFAICS the python snippet is the same in both cases? Well one of them uses tabs instead of spaces AFAICT :) Can't we use `find -maxdepth 1 -type f -iname '*ovirt-*.repo' -printf '%f\n'` instead?

Reply:
I'm afraid that this will be too broad of a match, and may potentially cover some repos we don't want to be activated by default.
So it's better to keep the matching pattern more focused, even at the price of running the search twice.